website/content/docs/concepts/stateful-deployments.mdx

```diff
@@ -0,0 +1,58 @@
+---
+layout: docs
+page_title: Stateful Deployments
+description: |-
+  Learn how Nomad handles Stateful Deployments.
+---
+
+# Stateful Deployments
+
+Nomad Stateful Deployments allow users to effectively bind their jobs to host
+[volumes]. On a lower level, this means taking Nomad's basic unit of workload
+placement—the [allocation]—and making it possible for users to have allocations
+that are ["sticky"] to volume IDs.
+
+<Note>
+Stateful Deployments support only Host Volumes. For CSI Volumes, we have <a
+href="/nomad/docs/job-specification/volume#per_alloc">per_alloc</a> property
+that serves a similar purpose.
+</Note>
+
+Stateful Deployments work on a per-task-group basis, that is, each task group
+can indicate that a host volume it requires must be "sticky." This is done by
+setting the parameter in the jobspec:
+
+```hcl
+job "app" {
+  group "example" {
+    # ...
+    volume "example" {
+      type      = "host"
+      source    = "ca-certificates"
+      read_only = true
+      sticky    = true
+    }
+    # ...
+  }
+}
+```
+
+If a volume is set to be sticky, during the deployment the scheduler adds its
+ID to the allocation that uses it. Stateful deployments require that any
+allocation that has the new VolumeID field set will only be (re)scheduled on
```

Comment:
Didn't we drop the "volume ID" field on the allocs? Should this discuss the claims object instead?